client/src/components/Checkout/Checkout.js

```diff
@@ -61,12 +75,66 @@ function Checkout() { // pass in props from the shopping cart page, such as item
                                         </div>
                                     </div>
                                 </div>
+                            </div>)}
+                        </div>
+
+                        <div className="card">
+                            <div className="dropdown-header" onClick={toggle1}>
+                                <h4 className="dropdown-text">Delivery option</h4>
+                                <span className="dropdown-selection">{selected1 ? '-' : '+'}</span>
                             </div>
+                            {selected1 && (<div className="wrapper">
+                                <div className="row">
+                                    <div className="col-md-6">
+                                        <div className="info-group mb-3">
+                                            <label>First name</label>
+                                            <input type="text" className="info-input" name="firstname" />
+                                        </div>
+                                    </div>
+                                    <div className="col-md-6">
+                                        <div className="info-group mb-3">
+                                            <label>Last name</label>
+                                            <input type="text" className="info-input" name="firstname" />
+                                        </div>
+                                    </div>
+                                    <div className="col-md-6">
+                                        <div className="info-group mb-3">
+                                            <label>Phone number</label>
+                                            <input type="text" className="info-input" name="firstname" />
+                                        </div>
+                                    </div>
+                                    <div className="col-md-12">
+                                        <div className="info-group mb-3">
+                                            <label>Address</label>
+                                            <input type="text" className="info-input" name="firstname" />
+                                        </div>
+                                    </div>
+                                    <div className="col-md-4">
+                                        <div className="info-group mb-3">
```

Comment:
Also, for enhancement we can try adding drop downs showing list of Cities, provinces etc.